addons/mail/static/src/xml/chatter.xml

```diff
@@ -99,7 +99,7 @@
 
         <!-- Add a button to open the full composer dialog -->
         <t t-jquery=".o_composer_button_add_attachment" t-operation="after">
-            <button tabindex="6" class="btn btn-secondary fa fa-expand o_composer_button_full_composer" type="button" title="Full composer" aria-label="Full composer"/>
+            <button tabindex="6" class="btn btn-secondary fas fa-expand o_composer_button_full_composer" type="button" title="Full composer" aria-label="Full composer"/>
```

Comment:
The new icon for `fa-expand` is actually very different, and it doesn't really convey the same meaning.
To validate with PO if we want a visual change here. I think the corresponding new icon to the old one is `expand-alt` now.

Reply:
Indeed they are quite different, oddly enough it is not listed in the changed icon list on the FA website. There are 10 matches in the source code, fa-compress is the opposite icon.

Reply:
As suggested by Seb, you can use `expand-alt`